packages/@aws-cdk/aws-logs/lib/log-retention.ts

```diff
@@ -152,6 +174,17 @@ class LogRetentionFunction extends Construct implements cdk.ITaggable {
       resources: ['*'],
     }));
 
+    role.addToPrincipalPolicy(new iam.PolicyStatement({
+      actions: ['logs:DeleteLogGroup', 'logs:DeleteLogStream'],
+      //Only allow deleting the specific log group.
+      resources: [cdk.Stack.of(this).formatArn({
```

Comment:
The resource for `logs:DeleteLogGroup` is correct, but not for `logs:DeleteLogStream`. See [Actions, resources, and condition keys for Amazon CloudWatch Logs
](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazoncloudwatchlogs.html)

Reply:
I modified the resource for logs:DeleteLogStream, but I am not sure if it is right. Could you please check it? I tried to deploy and destroy the stack in integ.log-retention.ts manually, it seems that the loggroup and relative log streams can be deleted even if I don't set the action for 'logs:DeleteLogStream'.